src/translib/phy_intf.go

```diff
@@ -382,6 +384,40 @@ func (app *IntfApp) processUpdatePhyIntfVlanAdd(d *db.DB) error {
 		if err != nil {
 			return errors.New("Updating VLAN table with member ports failed")
 		}
+		// Enable STP on L2 intefaces
+		enableStpOnInterfaceVlanMembership(d, stpInterfacesList)
+	}
+	return err
+}
+
+/* Adding member to LAG requires adding new entry in PORTCHANNEL_MEMBER Table */
+func (app *IntfApp) processUpdatePhyIntfLagAdd(d *db.DB) error {
```

Comment:
After this change the function "processUpdatePhyIntfLagAdd" is repeated twice in this file, can you please correct it.

Reply:
@Tejaswi-Goel  I see it. I'm wondering why it did not get flagged as a conflict. Are there any other changes added twice ?

Reply:
I did a manual diff between broadcom_sonic and brcm_poc1101, and those were the only diffs i notices (processUpdatePhyIntfLagAdd() added twice). Still investigating why the git merge didn't flag it. In the interim, i raised a PR to fix it (https://github.com/project-arlo/sonic-mgmt-framework/pull/249)